backend/src/main/java/com/woowacourse/levellog/prequestion/application/PreQuestionService.java

```diff
@@ -0,0 +1,121 @@
+
+    @Transactional
+    public Long save(final PreQuestionDto request, final Long levellogId, final Long memberId) {
+        final Levellog levellog = getLevellog(levellogId);
+        final Member questioner = getMember(memberId);
+
+        validateSameTeamMember(levellog.getTeam(), questioner);
+
+        return preQuestionRepository.save(request.toEntity(levellog, questioner))
+                .getId();
+    }
+
+    public PreQuestionDto findById(final Long preQuestionId, final Long levellogId, final Long memberId) {
+        final PreQuestion preQuestion = getPreQuestion(preQuestionId);
+        final Levellog levellog = getLevellog(levellogId);
+        final Member questioner = getMember(memberId);
+
+        validateLevellog(preQuestion, levellog);
+        validateMyQuestion(preQuestion, questioner);
+
+        return PreQuestionDto.from(preQuestion);
+    }
+
+    @Transactional
+    public void update(final PreQuestionDto request, final Long preQuestionId, final Long levellogId,
+                       final Long memberId) {
+        final PreQuestion preQuestion = getPreQuestion(preQuestionId);
+        final Levellog levellog = getLevellog(levellogId);
+        final Member questioner = getMember(memberId);
+
+        validateLevellog(preQuestion, levellog);
+        validateMyQuestion(preQuestion, questioner);
+
+        preQuestion.update(request.getPreQuestion());
+    }
+
+    @Transactional
+    public void deleteById(final Long preQuestionId, final Long levellogId, final Long memberId) {
+        final PreQuestion preQuestion = getPreQuestion(preQuestionId);
+        final Levellog levellog = getLevellog(levellogId);
+        final Member questioner = getMember(memberId);
+
+        validateLevellog(preQuestion, levellog);
+        validateMyQuestion(preQuestion, questioner);
+
+        preQuestionRepository.deleteById(preQuestion.getId());
+    }
+
+    private PreQuestion getPreQuestion(final Long preQuestionId) {
+        return preQuestionRepository.findById(preQuestionId)
+                .orElseThrow(() -> new PreQuestionNotFoundException(
+                        "사전 질문이 존재하지 않습니다. preQuestionId : " + preQuestionId));
+    }
+
+    private Levellog getLevellog(final Long levellogId) {
+        return levellogRepository.findById(levellogId)
+                .orElseThrow(() -> new LevellogNotFoundException("레벨로그가 존재하지 않습니다. levellogId : " + levellogId));
+    }
+
+    private Member getMember(final Long memberId) {
+        return memberRepository.findById(memberId)
+                .orElseThrow(() -> new MemberNotFoundException("멤버가 존재하지 않음 [memberId : " + memberId + "]"));
+    }
+
+    private void validateLevellog(final PreQuestion preQuestion, final Levellog levellog) {
+        if (!preQuestion.isSameLevellog(levellog)) {
+            throw new InvalidFieldException(
+                    "입력한 levellogId와 사전 질문의 levellogId가 다릅니다. 입력한 levellogId : " + levellog.getId());
+        }
```

Comment:
```suggestion
        preQuestion.validate(levellog);
```
이런식으로 책임을 도메인에게 넘겨줄 수 있을거 같습니다!

Reply:
감사합니다 배워갑니당